docs/docs-content/palette-cli/commands/fips-validate.md

```diff
@@ -0,0 +1,341 @@
+
+The `fips-validate` command exposes the following subcommands:
+
+- [`images`](#images): Validate the FIPS compliance of the images in your clusters.
+
+- [`services`](#services): Validate the FIPS compliance of the service endpoints in your clusters.
+
+- [`clean`](#clean): Remove the FIPS validation resources from your clusters.
+
+## Prerequisites
+
+The `fips-validate` command requires the following prerequisites:
+
+- Ensure you have access to the kubeconfig file for the cluster that you want to validate. The file needs to be
+  accessible from the machine where you issue the `fips-validate` command. Refer to the
+  [Set up Kubectl](../../clusters/cluster-management/palette-webctl.md#access-cluster-with-cli) guide to learn how to
+  get the kubeconfig file for your cluster.
+
+- Sufficient permissions to create a new namespace and deploy resources in the namespace. We recommend using an elevated
+  ClusterRole such as [_cluster-admin_](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)
+  to ensure that the command can create the necessary resources.
+
+- The Kubernetes cluster must be accessible from the machine where you issue the `fips-validate` command. Ensure that
+  the kubeconfig file is correctly configured to access the cluster.
+
+- The Kubernetes cluster must have internet access to download the images when using the `images` subcommand. Private
+  image registries are not supported.
+
+## Images
+
+The `images` subcommand validates the FIPS compliance of the images in your clusters. The command scans all the images
+of active containers. The image is downloaded from the source and unpacked through [Luet](https://luet.io/docs/) and all
+files that are Executable and Linkable Format binaries are scanned. The scan checks if the binaries are compiled with
+FIPS compliant cryptographic libraries.
+
+To enable the scan, the command deploys resources into a Kubernetes namespace named `vertex-ns`, unless otherwise
+specified, in the target cluster. Upon completion of the scan, the command prints the results of the scan to the
+terminal and cleans up the resources from the cluster.
+
+The `images` subcommand accepts the following flags:
+
+| **Short Flag** | **Long Flag** | **Description**                                                                                        | **Type** |
+| -------------- | ------------- | ------------------------------------------------------------------------------------------------------ | -------- |
+| `-n`           | `--namespace` | The namespace in which to deploy the resources for the FIPS scan. The default namespace is `vertex-ns` | string   |
+| `-f`           | `--out`       | The output file path to save the scan results. The default output is the terminal.                     | string   |
+| `-h`           | `--help`      | Display the help message for the `images` subcommand.                                                  | boolean  |
+
+### Limitations
```

Comment:
I think all the limitations could be put under a single `## Limitations` section, which would be cleaner.